controllers/todo.js

```diff
@@ -5,36 +5,192 @@ const { ToDo, Token } = require("../models");
 // HINT: Create a middleware for above :)
 
 const getAllToDo = async (req, res) => {
+    
+  const user = req.user
+  ToDo.find({$or: [{createdBy:user._id},{collaborators:user._id}]},{title:1,createdBy:1},function(err,foundTodos){
+    if(!err && foundTodos.length > 0){
+
+      let todosOfUser=[]
+      let todosCollaboratedTo=[]
+      foundTodos.forEach(function(todo){
+        if(todo.createdBy==user.id){
+          todosOfUser.push({"id":todo._id,"title":todo.title})
+        }else{
+          todosCollaboratedTo.push({"id":todo._id,"title":todo.title})
+        }
+      })
+      res.status(200).json({
+        "todosOfUser":todosOfUser,
+        "todosCollaboratedTo":todosCollaboratedTo
+      })
+    }else{
+      res.status(200).json([])
+    }
+  
+  })
+
   // Get the token in header.
   // Use the token to get all the ToDo's of a user
 };
 
 const createToDo = async (req, res) => {
+
   // Check for the token and create a todo
   // or throw error correspondingly
+  const {title} = req.body
+  const user = req.user
+  console.log
+  if(title.trim()==""){  //Checking if title is empty string(Only white space)
```

Comment:
`""` is treated as `false` value in javascript. So, there is no need for equal sign. 
Better would be 
```js
if(!title.trim()){
//logic here
}
```